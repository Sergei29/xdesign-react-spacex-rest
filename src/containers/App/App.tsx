import React from "react";

function App() {
  return <div className="App">sPaceX launches list.</div>;
}

export default App;
