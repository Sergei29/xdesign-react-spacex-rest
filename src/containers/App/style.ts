import styled from "styled-components";

export const AppContainer = styled.div`
  max-width: 90%;
  margin: 32px auto 0 auto;
  position: relative;
  min-height: 100vh;
  padding-bottom: 0.25rem;
  font-family: "Brandon Grotesque", Arial, Helvetica, sans-serif;
`;
