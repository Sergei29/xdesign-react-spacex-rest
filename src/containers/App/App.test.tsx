import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import App from "./App";

describe("Application container", () => {
  let wrapper: ShallowWrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it("should render", () => {
    expect(wrapper.length).toEqual(1);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
