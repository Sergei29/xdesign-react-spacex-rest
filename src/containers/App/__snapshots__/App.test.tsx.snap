// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application container should match snapshot 1`] = `ShallowWrapper {}`;
